apps/champions/test/battle_test.exs

```diff
@@ -745,6 +745,152 @@ defmodule Champions.Test.BattleTest do
                  maximum_steps: maximum_steps
                ).result
     end
+
+    test "Self Heal" do
+      # This test pairs a self healing unit with 10 health, against a damage dealing unit that makes 5 damage for each attack,
+      # the second one attacks 3 times, but since the healer unit will heal itself for the same amount the same number of times,
+      # the battle will end on a timeout.
+      maximum_steps = 15
+      skills_cooldown = 5
```

Comment:
```suggestion
      skills_cooldown = 5
      maximum_steps = (skills_cooldown + 1) * 3
```

Maximum_steps needs to be 18 for a 5-step cooldown skill to hit. This is because for every 5 steps of cooldown we get 1 step of action, and then the loop starts again

Reply:
Done!